lib/static/components/controls/menu-bar.jsx

```diff
@@ -33,7 +33,7 @@ class MenuBar extends Component {
         }
 
         return (
-            <div className="menu-bar" data-test-id='menu-bar'>
+            <div className="menu-bar" data-qa='menu-bar'>
```

Comment:
Gravity UI uses `data-qa` attribute and in some places it's not configurable. So I decided to use `data-qa` everywhere in the app for consistency.